common/app/views/fragments/header.scala.html

```diff
@@ -46,7 +46,15 @@
                                         We're funded by <span class="cta-bar__aus-moment-count"></span>
                                         readers across Australia.
                                         <br/>
-                                        Thank you for supporting us
+                                        <a class="cta-bar__cta cta-bar__aus-map-cta hide-until-tablet"
+                                        data-link-name="nav2 : aus-map-cta"
+                                        data-edition="@{
+                                            editionId
+                                        }"
+                                        href="https://support.theguardian.com/aus-2020-map?acquisitionData=%7B%22source%22%3A%22GUARDIAN_WEB%22%2C%22componentType%22%3A%22ACQUISITIONS_HEADER%22%2C%22componentId%22%3A%22Aus_Moment_2020_map_header%22%2C%22campaignCode%22%3A%22Aus_Moment_2020_map_header%22%7D&INTCMP=Aus_Moment_2020_map_header">
```

Comment:
I'm not sure this needs the `acquisitonData=` param. Plus `INTCMP` should probably be something like `Aus_moment_2020_frontend_header`

Reply:
good point, I blindly copied this from the trello card but it's not right.
At the moment there's no tracking in this page

Reply:
I've removed the `acquisitonData` and used your suggested INTCMP
We've also just added ophan to the map page